app/components/primer/beta/border_box/header.rb

```diff
@@ -5,8 +5,10 @@ module Beta
     class BorderBox
       # `BorderBox::Header` is used inside `BorderBox` to render its header slot.
       #
-      # @accessibility When using `header.title`, set `tag` to one of `h1`, `h2`, `h3`, etc. based on what is appropriate for the page context. <%= link_to_heading_practices %>
+      # @accessibility When using `header.with_title`, set `tag` to one of `h1`, `h2`, `h3`, etc. based on what is appropriate for the page context. <%= link_to_heading_practices %>
       class Header < Primer::Component
+        status :beta
```

Comment:
i was previously told sub components don't need a `status`, but i found that they show up as `alpha` status in `static/statuses.json` if they don't have one specifically set. so i think we should include them